### DISCUSSION TITLE
fails to update Paperless ngx

### DISCUSSION THREAD

**Discussion by knorst1** [tteck/Proxmox#3131] [Help Needed]
while updating to the newest paperless ngx version it fails with the following line. I used the Proxmox helper script to initiate the update procedure.
it says: [ERROR] in line 83: exit code 0: while executing command /usr/bin/python3 manage.py migrate &> /dev/null
if i restart the lcx it says it has the newest version 2.9.0.?
was the update successfull or not?
greetings

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/2925
